scipy/optimize/_linprog_highs.py

```diff
@@ -0,0 +1,186 @@
+    time_limit : float
+       The maximum time alotted to solve the problem; default 1s.
+    presolve : bool
+        Set to ``False`` if presolve is to be disabled; default ``True``.
+    parallel : bool
+        Set to ``True`` to enable parallelization; default ``False``.
+    unknown_options : dict
+        Optional arguments not used by this particular solver. If
+        `unknown_options` is non-empty, a warning is issued listing all
+        unused options.
+
+    Returns
+    -------
+    sol : dict
+        A dictionary consisting of the fields:
+
+            x : 1D array
+                The values of the decision variables that minimizes the
+                objective function while satisfying the constraints.
+            fun : float
+                The optimal value of the objective function ``c @ x``.
+            slack : 1D array
+                The (nominally positive) values of the slack variables,
+                ``b_ub - A_ub @ x``.
+            con : 1D array
+                The (nominally zero) residuals of the equality constraints,
+                ``b_eq - A_eq @ x``.
+            success : bool
+                ``True`` when the algorithm succeeds in finding an optimal
+                solution.
+            status : int
+                An integer representing the exit status of the algorithm.
+
+                ``0`` : Optimization terminated successfully.
+
+                ``1`` : Iteration limit reached.
+
+                ``2`` : Problem appears to be infeasible.
+
+                ``3`` : Problem appears to be unbounded.
+
+                ``4`` : Numerical difficulties encountered.
+
+            nit : int
+                The total number of iterations performed in all phases.
+            message : str
+                A string descriptor of the exit status of the algorithm.
+
+    """
+
+    _check_unknown_options(unknown_options)
+
+    messages = ["Optimization terminated successfully.",
+                "Iteration limit reached.",
+                "The problem appears infeasible, as the phase one auxiliary "
+                "problem terminated successfully with a residual of {0:.1e}, "
+                "greater than the tolerance {1} required for the solution to "
+                "be considered feasible. Consider increasing the tolerance to "
+                "be greater than {0:.1e}. If this tolerance is unnaceptably "
+                "large, the problem is likely infeasible.",
+                "The problem is unbounded, as the simplex algorithm found "
+                "a basic feasible solution from which there is a direction "
+                "with negative reduced cost in which all decision variables "
+                "increase.",
+                "Numerical difficulties encountered; consider trying "
+                "method='interior-point'.",
+                ]
+
+    c, A_ub, b_ub, A_eq, b_eq, bounds, x0 = lp
+    lb, ub = bounds.T                       # separate bounds, TODO: C-cntgs
```

Comment:
@mckib2 do `lb` and `ub` need to be passed into the wrapper as C-contiguous arrays?

Reply:
Yes, the wrappers expect arrays to be given c-contiguous